python_modules/libraries/dagster-duckdb/dagster_duckdb/io_manager.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+from typing import cast
+
+import duckdb
+import pandas as pd
+
+from dagster import Field, IOManager, InputContext, OutputContext
+from dagster import _check as check
+from dagster import io_manager
+from dagster._seven.temp_dir import get_system_temp_directory
+from dagster._utils.backoff import backoff
+
+
+class DuckDBCSVIOManager(IOManager):
```

Comment:
should this eventually follow this sort of pattern: https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/python_modules/libraries/dagster-snowflake/dagster_snowflake/db_io_manager.py?L69:7 ?

Reply:
yeah i was thinking that as well. i didn't want to sink a ton of time into getting this up, but i might see how far i can get in a couple hours on moving the io manager to that pattern